index.html

```diff
@@ -145,9 +145,17 @@ <h2>Extensions to the <dfn data-cite="HTML5#the-window-object">Window</dfn> inte
-      <p class="note">Generally the events that are considered to be continous are <dfn data-cite="uievents#event-type-mousemove">mousemove</dfn>, <dfn data-cite="uievents#event-type-wheel">wheel</dfn> and <dfn data-cite="touch-events#the-touchmove-event">touchmove</dfn> all of which are specified in such a way where the browser is encouraged to fire the events at a rate specific for the browser and platform. Events that are introduced in the future and similar should also probably be considered to be continous.</p>
+      <h2>ContinuousEvents</h2>
+      <p class="note">The distinction between <a>ContinuousEvents</a> and other event types exists in order to prevent script from yielding too often. Under normal circumstances a user moving their pointer around on a document wouldn’t expect that to have any effect on performance so by default these events are excluded from <a data-link-for="Scheduling">isInputPending</a>. However, there may be times when a document wants to yield for these events.</p>
+      <section>
+        <h3><dfn data-lt="ContinuousEvents">ContinuousEvent</dfn></h3>
+        <p>An event MUST be considered a <code>ContinuousEvent</code> if it is a <dfn data-cite="dom#dom-event-istrusted">trusted event</dfn> of type <dfn data-cite="uievents#event-type-mousemove">mousemove</dfn>, <dfn data-cite="uievents#event-type-wheel">wheel</dfn>, <dfn data-cite="touch-events#the-touchmove-event">touchmove</dfn>, <dfn data-cite="html#dragevent">drag</dfn>, or <dfn data-cite="pointerevents#the-pointermove-event">pointermove</dfn> or a child of any of those. A user agent MAY consider other trusted events that are of a specific type and are children of <a>UIEvent</a> or <a>TouchEvent</a> to be a <code>ContinuousEvent</code> as long as those specific types are consistent for that user agent and that all events of those types or children of those types are considered to be <code>ContinuousEvents</code>.</p>
+        <p class="note">All of event types that are considered to be <code>ContinuousEvents</code> are specified in such a way where they are fired successively, and the browser is encouraged to fire the events at a rate specific for the browser and platform. The language about a user agent being able to include other event types is there so browsers that have nonstandard events that are defined in a similar manner may also be considered to be <code>ContinuousEvents</code>.</p>
+      </section>
+      <section>
+        <h3><dfn>ContinuousEventTask</dfn></h3>
+        <p>A <a>task</a> MUST be considered to be a <code>ContinuousEventTask</code> if it will <a>dispatch</a> an event that is a <a>ContinuousEvent</a>.</p>
```

Comment:
nit: be consistent between "considered to be a" vs "considered a" (not sure which ones is better).